SoMRandomizer/processing/common/structure/Tileset16.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+
+namespace SoMRandomizer.processing.common.structure
+{
+    /// <summary>
+    /// Tileset containing 384 16x16 Tiles - 192 background and 192 foreground.
+    /// </summary>
```

Comment:
did you want to tag yourself as author consistently with the rest of my files?  there didn't seem to be a standard tag for it, so i added it in a remarks block